libjolie/src/jolie/lang/parse/SemanticVerifier.java

```diff
@@ -386,6 +387,9 @@ public void validate()
 		}
 	}
 
+	@Override
+	public void visit( ReThrowStatement n ) {}
```

Comment:
The only way to fix that is to define a primitive that checks whether we're doing fault recovery that we can statically determine.

But I don't see a good use of that. Why would you define a procedure that either recovers something or proceeds normally? You should just install something different. I generally dislike using bookkeeping variables since we have such a powerful mechanism as dynamic fault handling already. If anything, I would like people _not_ to use bookkeeping variables. But maybe there's something you cannot write without using them?